README.md

```diff
@@ -790,6 +790,7 @@ A GitHub conflict is when people make changes to the same area or line in a file
   - [Molly Vanessa](https://github.com/molly460)
   - [Mridul Birla](https://github.com/Mridulbirla13)
   - [Mrinaal Arora](https://github.com/aroramrinaal)
+  - [Mahendra Bishnoi](https://github.com/MahendraBishnoi29)
```

Comment:
Please move your name to line 736, after Mahaveer Singh Barhath.
Let us know if you need any help doing that! 🙂 


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not maintainable

Reply:
how do i change this now?

Reply:
hey thanks for letting me know I have changed this, is it ok now?